Makefile

```diff
@@ -40,22 +42,18 @@ db: python
 
 .PHONY: dev
 dev: build/manifest.json python
-	@tox -qe dev --run-command 'honcho start ${processes}'
+	@tox -qe dev
+
+.PHONY: supervisor
+supervisor: python
+	@tox -qe dev --run-command 'supervisorctl -c conf/supervisord-dev.conf $(command)'
```

Comment:
`supervisorctl` allows you to do what `make assets` and `make web` used to do (run just the assets or web process and connect to its stdin/out directly so you can use `pdb`) as well as many other things.

Demo:

```terminal
$ # Equivalent to `make web` previously, pdb works.
$ make supervisor command='fg web'
==> Press Ctrl-C to exit <==
... (output from gunicorn)

$ # Or just launch the interactive supervisorctl shell. It has builtin help.
$ make supervisor
assets                           RUNNING   pid 2479148, uptime 0:12:14
logger                           RUNNING   pid 2479147, uptime 0:12:14
web                              RUNNING   pid 2479149, uptime 0:12:14
lms> help

default commands (type help <topic>):
=====================================
add    exit      open  reload  restart   start   tail   
avail  fg        pid   remove  shutdown  status  update 
clear  maintail  quit  reread  signal    stop    version
```